src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,50 @@
+import { Loader } from '../Loader';
+import { PeopleFilters } from '../PeopleFilters';
+import { PeopleTable } from '../PeopleTable';
+
+import { usePeople } from '../../hooks/usePeople';
+
+export const PeoplePage = () => {
+  const { people, error, loading, filteredPeople } = usePeople();
+
+  const succesfullyLoadedStatus = !!people.length && !loading;
```

Comment:
`succesfullyLoadedStatus `is descriptive but a bit verbose. Consider renaming it to `isLoaded `for simplicity.